w06h02/test/pgdp/ds/MultiStackTest.java

```diff
@@ -76,7 +76,7 @@ void pushOutOfCapacityShouldCreateNewStack() {
         stack.push(value);
         final var next = getNext(getHead());
         final var mem = getMem(next);
-        assertTrue(arrayContains(mem, value));
+        assertEquals(mem[0], value);
```

Comment:
Where does the exercise say that values have to be added to the front of the mem array? A valid solution would also be to add new values to the end of the array and make the stack pointer point there.

Reply:
Yeah you're right

Reply:
I just thought about implementing that... I wont do it. feel free to do it yourself but keep in mind that that you would have to change all other `mem` tests where `mem.length < 1` as well.

Reply:
That's fine. Most people won't implement it that way anyways.